codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/generators/ParserGeneratorBase.java

```diff
@@ -301,9 +300,9 @@ public void render(FileManifest fileManifest) {
     }
 
     protected void renderParsers() {
-        TreeSet<Shape> shapesToRender = CodegenUtils.getAlphabeticalOrderedShapesSet();
-        TreeSet<Shape> eventStreamEventsToRender = CodegenUtils.getAlphabeticalOrderedShapesSet();
-
+        Comparator<Shape> comparator = Comparator.comparing(o -> o.getId().getName() + " " + o.getId());
```

Comment:
Why this change? The codegenutils helper method for this seems cleaner than implementing a comparator for each one?